common/scala/src/main/scala/org/apache/openwhisk/core/containerpool/logging/DockerToActivationLogStore.scala

```diff
@@ -68,8 +68,16 @@ class DockerToActivationLogStore(system: ActorSystem) extends LogStore {
   override val containerParameters = Map("--log-driver" -> Set("json-file"))
 
   /* As logs are already part of the activation record, just return that bit of it */
-  override def fetchLogs(activation: WhiskActivation, context: UserContext): Future[ActivationLogs] =
-    Future.successful(activation.logs)
+  override def fetchLogs(namespace: String,
+                         activationId: String,
```

Comment:
we have a type `ActivationId` instead of `string` - is it more convenient to use string here?

Reply:
👍  just noticed the `toString` override in the `ActivationId` class